src/open_inwoner/accounts/models.py

```diff
@@ -397,8 +397,8 @@ def get_contact_message_url(self) -> str:
         return f"{url}#messages-last"
 
     def get_contact_type_display(self) -> str:
-        choice = ContactTypeChoices.get_choice(self.contact_type)
-        return choice.label
+        choices_mapping = dict(ContactTypeChoices.choices)
+        return choices_mapping[self.contact_type]
```

Comment:
Minor, but I think it's better to do something like this

```suggestion
        choice = getattr(ContactTypeChoices, self.contact_type)
        return choice.label
```

```python
>>> timeit.timeit(lambda: dict(FooChoices.choices)["foo"], number=100000)
0.2956498809999175
>>> timeit.timeit(lambda: getattr(FooChoices, "foo").label, number=100000)
0.05368091000127606
```